### DISCUSSION TITLE
Visbug extension triggers TypeError: Cannot read properties of undefined (reading 'find') when resize.

### DISCUSSION THREAD

**Discussion by samyogdhital** [vercel/next.js#47230] [Help]
### I am not sure if this the bug with Next.js or Visbug extension. But when I am in development this error always appears.
![tinywow_tinywow_2023-03-17 12-35-25_16723831_16723954](https://user-images.githubusercontent.com/91406792/225834523-086f0dc6-9289-4f37-918f-eddfb87ecc37.gif)

### Additional information

```js
If you need any more information, I am happy to provide to you.
```

**Comment by icyJoseph** [comment]
Hi,

The Next.js error overlay catches, all errors that happen in your browser runtime. You can see here the handlers they use:

- https://github.com/vercel/next.js/blob/canary/packages/react-dev-overlay/src/client.ts

This is so that you can catch, not only React rendering errors, but errors in your entire app. Unfortunately, 3rd party errors might leak into it... This very much looks like Visbug error. 

Perhaps this one? https://github.com/GoogleChromeLabs/ProjectVisBug/issues/17#issuecomment-1410621182

Try to update your extension.

**Comment by samyogdhital** [comment]
Confirmed and it has something to do with the Visbug extension itself. Thanks @icyJoseph. Closing this discussion.
